_posts/2020-08-05-#19160.md

```diff
@@ -7,17 +7,47 @@ authors: [ryanofsky]
 components: ["build system"]
 host: ryanofsky
 status: upcoming
-commit:
+commit: 0602f87
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+These review club notes are for PR [#19160 multiprocess: Add basic spawn and IPC support](https://github.com/bitcoin/bitcoin/pull/19160) ([branch](https://github.com/ryanofsky/bitcoin/commits/pr/ipc-echo.7)), focused on the second eponymous commit: [`aa4d626`](https://github.com/ryanofsky/bitcoin/commit/aa4d626db1628d5e29572eccfd7fa2d99edeeb60).
+
+The key things this commit adds are an [`IpcProcess::spawn()`](https://github.com/ryanofsky/bitcoin/blob/pr/ipc-echo.7/src/interfaces/ipc.h#L24-L26) method which a bitcoin parent process can call to spawn a new child process, and an [`IpcProcess::serve()`](https://github.com/ryanofsky/bitcoin/blob/pr/ipc-echo.7/src/interfaces/ipc.h#L31-L33) method, which a bitcoin child process can call [after](https://github.com/ryanofsky/bitcoin/blob/pr/ipc-echo.7/src/bitcoind.cpp#L180-L185) [startup](https://github.com/ryanofsky/bitcoin/blob/pr/ipc.120/src/bitcoin-wallet.cpp#L82-L88) to communicate back to its parent process.
+
+In followup PR [#10102 Multiprocess bitcoin](https://github.com/bitcoin/bitcoin/pull/10102), this functionality is used so a `bitcoin-gui` process can spawn a `bitcoin-node` process, and a `bitcoin-node` process can spawn a `bitcoin-wallet` process, and gui, node, and wallet functionality can run in separate processes that are protected from each other. In further followups [#19460 Add bitcoin-wallet -ipcconnect option](https://github.com/bitcoin/bitcoin/pull/19460) and [#19461 Add bitcoin-gui -ipcconnect option](https://github.com/bitcoin/bitcoin/pull/19461), more flexibility is added so node, wallet, and GUI processes can be started and stopped independently.
```

Comment:
Was this the original parent PR that has been updated to be the next step after 19160?

Throughout the document: s/followup/follow-up/